app/views/harvest_schedules/_enrichments.html.erb

```diff
@@ -2,7 +2,7 @@
   <% if harvest_schedule.parser %>
     <% if harvest_schedule.parser.enrichment_definitions(environment).any? %>
 
-      <%= custom_form_with model: harvest_schedule, local: true do |f| %>
+      <%= custom_fields_for :harvest_schedule, local: true do |f| %>
```

Comment:
@Reinkaos What was the change that made this work. Something about nested forms was it?

Reply:
Hey @daveharris, yeah we were using this  `custom_form_with_models` which generates a whole new form tag, so we ended up with a form inside of a form. Using fields_for generates the form fields without the form tag, so we don't end up with a nested form.